docs/src/main/asciidoc/opentelemetry.adoc

```diff
@@ -138,6 +137,33 @@ All configurations have been updated from `quarkus.opentelemetry.\*` -> `quarkus
 The legacy configurations are now deprecated but will still work during a transition period.
 ====
 
+=== Disable all or parts of the OpenTelemetry extension
```

Comment:
I am wondering if we should keep a manual configuration list here. The config is auto-generated and included in the end of the document. Maybe a link is enough?

Reply:
I agree, but for some weird reason, some of the properties are not shown there and this section also provides some context. We have a lot of questions around disabling this and that.

Reply:
Which ones are not showing? The old ones? That is because we choose to remove them completely from the Config system. If we want to display them, then we need to add them again in the mapping classes.

Reply:
All except `quarkus.otel.sdk.disabled`, which is a runtime one... Should I submit a bug?